modules/kuberay-cluster/kuberay-autopilot-values.yaml

```diff
@@ -391,8 +207,10 @@ additionalWorkerGroups:
       gke-gcsfuse/memory-limit: 20Gi
       gke-gcsfuse/ephemeral-storage-limit: 20Gi
     nodeSelector:
+      cloud.google.com/compute-class: "Performance"
+      cloud.google.com/machine-family: "c3"
+      cloud.google.com/gke-ephemeral-storage-local-ssd: "true"
       iam.gke.io/gke-metadata-server-enabled: "true"
-      cloud.google.com/gke-accelerator: "nvidia-tesla-t4"
```

Comment:
dont we still need to pick L4 somewhere for AP?

Reply:
Yes, in the next worker group. For RAG, our Ray job will likely scale the gpuGroup. However, other applications may prefer CPU.